repos/fdbt-site/cypress_tests/cypress/support/index.ts

```diff
@@ -0,0 +1,189 @@
+import {
+    clickElementById,
+    clickElementByText,
+    getRandomNumber,
+    getHomePage,
+    throwInvalidRandomSelectorError,
+} from './helpers';
+import { enterPassengerTypeDetails, addGroupPassengerType } from './passengerTypes';
+import { addPurchaseMethod } from './purchaseMethods';
+import { addTimeRestriction } from './timeRestrictions';
+
+before(() => {
+    cy.log('index.ts was run');
+    getHomePage();
+    clickElementById('account-link');
+    clickElementByText('Passenger types');
+    randomlyAddPassengerTypes();
+    clickElementByText('Purchase methods');
+    randomlyAddPurchaseMethods();
+    clickElementByText('Time restrictions');
+    randomlyAddTimeRestrictions();
+    cy.log('Global Settings set up for BLAC');
+
+    getHomePage('scheme');
+    clickElementById('account-link');
+    clickElementByText('Passenger types');
+    randomlyAddPassengerTypes();
+    clickElementByText('Purchase methods');
+    randomlyAddPurchaseMethods();
+    clickElementByText('Time restrictions');
+    randomlyAddTimeRestrictions();
+    cy.log('Global Settings set up for scheme');
+});
+
+const randomlyAddPassengerTypes = (): void => {
+    cy.get(`[data-card-count]`).then((element) => {
+        const numberOfPassengers = Number(element.attr('data-card-count'));
+        cy.log(`There are ${numberOfPassengers} individuals/groups`);
+        if (numberOfPassengers > 1) {
+            cy.log('There is at least two passenger types');
+        } else {
+            const passengerType1 = {
+                type: 'child',
+                maxAge: 18,
+                name: 'Small People',
+            };
+            const passengerType2 = {
+                type: 'student',
+                documents: ['student_card'],
+                name: 'Test Students',
+            };
+            const passengerType3 = {
+                type: 'adult',
+                name: 'Big People',
+            };
+            const randomSelector = getRandomNumber(1, 2);
+            switch (randomSelector) {
```

Comment:
Just wondering if we need the complexity of the random chance or if we always just create the same, given that this will only run once anyway?

Reply:
We don't need it, it just widens what gets tested ever so slightly, very easy to just to do the last case for each GS setting if you'd prefer that

Reply:
Changed to just run the last of each one